### ISSUE TITLE
GitColony / BountyColony -- Divide and Conquer a single StandardBounty

### ISSUE THREAD

**Issue by austintgriffith** [JoinColony/colonyHackathon#39]
### Colony Hackathon Submission

GitColony / BountyColony 

---

A colony of developers and designers could form behind a single GitCoin or StandardBounties format bounty. I'm not sure if this fits the Colony model... Recently, I wanted to submit a large bounty to GitCoin that might require multiple workers with multiple disciplines. Then, let the community divide and conquer on the bounty. Dividing the issue myself seemed like micromanaging. I don't think there is a way to do this currently with a StandardBounty? I would love feedback on if this is even a proper use of Gitcoin+Colony. 

https://github.com/austintgriffith/GitColony

[Austin Griffith](https://austingriffith.com) @austingriffith austin@concurrence.io 

Would love any help or feedback on integrating StandardBounties and Colony.

**Comment by matth3us** [comment]
That sounds like an awesome project! It could unlock bigger bounties in GitCoin, and make it more meaningful for open source projects!  I don't know how you picture this in practice, though. Something like: open a bounty with something like a mark for "Colony project required", connect that bounty to a colony and let volunteers for that bounty to organize themselves?

**Comment by austintgriffith** [comment]
Yeah I think Colony would provide the platform for splitting, delegating, and rewarding smaller steps of progress toward a larger overall bounty. Still wondering if this is a good application for this hackathon and I would love feedback or help from the community.

**Comment by jrmoreau** [comment]
Thanks for posting @austintgriffith !

**Comment by austintgriffith** [comment]
Neat! Gitcoin.co launched cooperative bounties today. 
[https://medium.com/gitcoin/we-listened-announcing-project-types-965a02603559](https://medium.com/gitcoin/we-listened-announcing-project-types-965a02603559)

This address the issue I'm trying to work through, but now I'm wondering how the task delegation/rewarding happens. I will have to play around with the GitCoin interface to decide if Colony would still work well for this. Maybe a standard could still be built around StandardBounties+Colony? 

If anyone has more insight into this, please share :)

**Comment by jrmoreau** [comment]
@gichiba any thoughts on the delegation question?

**Comment by gichiba** [comment]
Yeah, I think in principle Colony's reputation would do the 'delegation' part quite well. I'd imagine it going down sorta like this: GitColony accepts work on a bounty. Tokens get minted equal to the bounty value. Members of the GitColony create and fund tasks with the tokens. When they complete the tasks, they get tokens and reputation in GitColony. They can then either sell the tokens back to the colony (this is not an explicit feature in colonyNetwork so you might have to think more about how the 'buyback' process would work), or keep them to continue earning rewards.

**Comment by austintgriffith** [comment]
@gichiba Thanks for the feedback! I've updated the original issue submission to reflect what we have discussed so far including a new Fulfillment contract. This contract would receive the Standard Bounty funds and pay them out to earners of the token native to the Colony. 

Does this look like a feasible hackathon project?
Is the Colony.io Beta UI available for hackathon participants? Can a dude get a login?
